substrate/frame/nfts/src/impl_nonfungibles.rs

```diff
@@ -362,6 +389,21 @@ impl<T: Config<I>, I: 'static> Mutate<<T as SystemConfig>::AccountId, ItemConfig
 			<Self as Mutate<T::AccountId, ItemConfig>>::clear_collection_attribute(collection, k)
 		})
 	}
+
+	fn clear_metadata(
+		who: &T::AccountId,
```

Comment:
```suggestion
		who: Option<&T::AccountId>,
```

Reply:
Done! Also changed the naming to `clear_item_metadata` as we did for the one above